source/MapPanel.cpp

```diff
@@ -1016,7 +1016,7 @@ void MapPanel::DrawTravelPlan()
 	bool hasEscort = false;
 	map<const Ship *, double> fuel;
 	for(const shared_ptr<Ship> &it : player.Ships())
-		if(!it->IsParked() && !it->CanBeCarried() && it->GetSystem() == flagship->GetSystem())
+		if(!it->IsParked() && (!it->CanBeCarried() || it.get() == flagship) && it->GetSystem() == flagship->GetSystem())
```

Comment:
> In theory, you could extend this idea to any fighter (not just the flagship) that's been modified for jump travel (and isn't able to be carried). But that would be a lot more work. ;)

Couldn't we do something like `!it->CanBeCarried || it->JumpNavigation().HasTravel()`, with `HasTravel()` being a new ShipJumpNavigation method that's shorthand for `HasHyperdrive() || HasJumpDrive()`?